piton/design/include/piton_system.vh

```diff
@@ -134,6 +136,11 @@
     `define PITON_FPGA_RST_ACT_HIGH
 `endif
 
+`ifdef XUPP3R_BOARD
```

Comment:
Was this necessary? Does it work without it? Just want to be sure.

Reply:
No, it doesn't